addon-test-support/audit-if.ts

```diff
@@ -27,10 +27,10 @@ function getUrlParameter(name: string) {
 export default function a11yAuditIf(
   contextSelector?: ElementContext | RunOptions | undefined,
   axeOptions?: RunOptions | undefined
-) {
+): Promise<void> {
```

Comment:
FWIW, I'd rather use PromiseLike :P

Reply:
Yes, we've had this conversation before. I can probably do that, as in this case we don't need any other functions in our promise usage.